### DISCUSSION TITLE
Getting drizzleConnection arguments inside an input key (with input plugin)

### DISCUSSION THREAD

**Discussion by olup** [hayes/pothos#1338] [Q&A] [answered]
Hello, I am diving in this very helpful library and playing with the drizzle plugin. Is there a way to get arguments for the drizzleConnection to have auto-generated argument types inside an input key ?

**Comment by hayes** [comment, marked as answer]
Assuming you are asking about something like the [`with-input`](https://pothos-graphql.dev/docs/plugins/with-input) plugin, I don't think there is currently a way to do this inline.

The best option is probably to just use `builder.inputObject` to create your input type separately, and then add an `input` arg to your connection args.
